scripts/monitoring/cron-send-create-app.py

```diff
@@ -65,6 +72,8 @@ def parse_args():
     parser.add_argument('--basename', default="test", help='base name, added to via openshift')
     parser.add_argument('--namespace', default="ops-health-monitoring",
                         help='namespace (be careful of using existing namespaces)')
+    parser.add_argument('--loopcount', default="36", 
```

Comment:
optional: would it be worth introducing all top magic numbers as params here? loop max, loop delay, no pod count max